ajax/libs/turn.js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "turn.js",
+  "version": "3",
+  "description": "The page flip effect for HTML5",
+  "filename": "turn.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/blasten/turn.js.git"
+  },
+  "keywords": [
+    "flipbooks",
+    "HTML5",
+    "CSS3"
+  ],
+  "author": "Emmanuel Garcia",
+  "license": "BSD",
```

Comment:
0.1 and 0.11 is under MIT
but v3 is under BSD
https://github.com/blasten/turn.js?files=1
http://turnjs.com/docs/Turn.js_3th_release_license_agreement_for_noncommercial_projects